scripts/main.js

```diff
@@ -6,6 +6,7 @@ Minecraft Pixel Art Maker
 /** Colours that are selected on creation of a new image form. */
 const default_palette = Array.from(Colours.keys()).join(' ');
 
+let counter = 0;
```

Comment:
This is probably better wrapped in a function along with the drag/drop event listener callbacks
- won't add to the global namespace
- will be separate for each image form instance